format-with-existing-config.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+# AWS Greengrass Nucleus Code Formatting Script
+# Uses existing codestyle/IntelliJ.xml configuration
+
+set -e
+
+# Colors for output
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+YELLOW='\033[1;33m'
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

YELLOW appears unused. Verify use (or export if used externally).